src/main.rs

```diff
@@ -29,6 +29,24 @@ fn notmain() -> Result<i32> {
     // NOTE `skip` the name/path of the binary (first argument)
     let raw_args = env::args().skip(1).collect::<Vec<_>>();
 
+    // If there's no argument provided, print the help message
+    if matches!(
+        raw_args
+            .iter()
+            .map(|s| s.as_str())
+            .collect::<Vec<&str>>()
+            .as_slice(),
+        [] | ["--help"] | ["-h"]
+    ) {
```

Comment:
This one should work:
```suggestion
    if let None | Some("--help" | "-h") = raw_args.first().map(String::as_str) {
```
(If it does not lets just leave it. But I kinda got nerd sniped here, hehe)